warp-core/src/main/scala/com/workday/warp/persistence/CorePersistenceAware.scala

```diff
@@ -291,15 +284,11 @@ trait CorePersistenceAware extends PersistenceAware {
         action <- tags.toList match {
           case Nil =>
             this.writeTestDefinitionTagQuery(TestDefinitionTagRow(Tables.nullId, idTestDefinition, nameRow.idTagName, value))
-          case (oldKey, oldValue) :: Nil if oldValue.equals(value) =>
+          case (oldKey, oldValue) :: Nil =>
```

Comment:
So we are allowing the test result to persist but we are retaining the old tag value?